### PR TITLE
Parse set values

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

**Comment by akofink** [comment]
Hmm... looking at this and your PR for fixes/subs makes me reconsider my position. It's clear that these lowercase elements are still quite complicated (lots of attributes and sometimes nested elements). I'm going to take another look at both PRs.

File: lib/openscap_parser/set_values.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'openscap_parser/set_value'
+
+module OpenscapParser
+  module SetValues
```
Comment (akofink) [inline lib/openscap_parser/set_values.rb]: Ok, so I think for this and other 'Selectors' (section 6.5.3 of the [XCCDF spec](https://csrc.nist.gov/publications/detail/nistir/7275/rev-4/final)) we should namespace them to reduce confusion. In almost all cases, we will want to be able to parse any of the selectors, not just `set-value`. What do you think about an `OpenscapParser::Selectors` module that includes `SetValues`? We don't need to implement the other selectors such as refine-value or refine-rule until we need them, but it allows us to add them easily later without changing the design.

File: lib/openscap_parser/profile.rb [hunk 2]
```diff
@@ -24,13 +24,6 @@ def selected_rule_ids
         @parsed_xml.xpath("select[@selected='true']/@idref").map(&:text)
     end
 
-    def set_values
-      @set_values ||= @parsed_xml.xpath("set-value") &&
-        @parsed_xml.xpath("set-value").map do |set_value|
-        [set_value['idref'], set_value.text]
-      end.to_h
-    end
-
```
Comment (akofink) [inline lib/openscap_parser/profile.rb]: :+1: it looks like profiles more often use `refine-value` than `set-value` anyway, though all of `select, set-complex-value, set-value, refine-value, refine-rule` are allowed.

File: lib/openscap_parser/set_value.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+require 'openscap_parser/xml_node'
+
+module OpenscapParser
+  class SetValue < XmlNode
+    def idref
```
Comment (akofink) [inline lib/openscap_parser/set_value.rb]: We've called this `id` in other modules.

**Comment by akofink** [comment]
Ok, overall I'm happy with this PR :+1: Scratch what I said earlier about uppercase XCCDF elements vs. lowercase (seems arbitrary anyway :upside_down_face: )

**Comment by xprazak2** [comment]
I renamed `idref` to `id` and added `Selectors` module.

**Pull request merged**